src/uiTests/JestTests/TestScripts/__tests__/app.spec.js

```diff
@@ -0,0 +1,10 @@
+import { mount } from '@vue/test-utils'
```

Comment:
keep test cases inside the TestScripts only instead of creating another folder __tests__